### PR TITLE
[Docs] Adding haproxy cfg file to the node file list.

### PR DESCRIPTION
**Pull request by rajkaramchedu**
### Test Plan

- - -
This change is [Reviewable](https://reviewable.io/reviews/aptos-labs/aptos-core/4966)

### CODE REVIEW

**Comment by rajkaramchedu** [comment]
Docs preview link: https://deploy-preview-4966--aptos-developer-docs.netlify.app/nodes/validator-node/operator/running-validator-node/run-validator-node-using-aws

**Review by Markuze — APPROVED** [review]

**Review by clay-aptos — APPROVED** [review]
Raj, you are fantastic! Thank you.

File: developer-docs-site/docs/nodes/node-files.md [hunk 1]
```diff
@@ -80,3 +98,12 @@ Fullnode means either a validator fullnode or a public fullnode.
   ```bash
   wget -O fullnode.yaml https://raw.githubusercontent.com/aptos-labs/aptos-core/mainnet/docker/compose/aptos-node/fullnode.yaml
   ```
+
+## haproxy-fullnode.cfg (fullnode only)
+
+- **Git repo:** `aptos-core`
+- **Git branch:** `main` on https://github.com/aptos-labs/aptos-core
+- **Command to download:**
+  ```bash
```
Comment (perryjrandall) [inline developer-docs-site/docs/nodes/node-files.md:107]: The branches are inconsistent here, sometimes mainnet, sometimes mainnet deployment sometimes main

Lets make sure

1) all files present on mainnet_deploytment
2) all the references here are mainnet_deployment if this is for mainnet

Is this document only for mainnet?

Comment (rajkaramchedu) [reply developer-docs-site/docs/nodes/node-files.md:107]: I was following your note in this Slack thread: https://aptos-org.slack.com/archives/C03EG004E56/p1665531016071549?thread_ts=1665529285.939749&cid=C03EG004E56
but I wasn't sure if I understood what you were saying. Hence my request to review this aspect. 

Documents will now be by default for mainnet. I will mention wherever testnet, devnet can be used. But for now, all mainnet.

File: developer-docs-site/docs/nodes/validator-node/operator/running-validator-node/using-aws.md [hunk 2]
```diff
@@ -158,8 +158,12 @@ This will download all the Terraform dependencies into the `.terraform` folder i
 
     This will create two YAML files in the `~/$WORKSPACE/$USERNAME` directory: `owner.yaml` and `operator.yaml`. 
 
-12. Download the genesis blob and waypoint for the network you want to connect to. See [Node Files](/nodes/node-files.md) for a full list of files you should download and the download commands. 
-
+12. Download the following files by following the download commands on the [Node Files](/nodes/node-files.md) page:
+    - `genesis.blob`
+    - `waypoint.txt`
+    - `haproxy.cfg`
+    - `haproxy-fullnode.cfg`
```
Comment (perryjrandall) [inline developer-docs-site/docs/nodes/validator-node/operator/running-validator-node/using-aws.md:165]: If this is aws we probably dont need haproxy files here

File: developer-docs-site/docs/nodes/validator-node/operator/running-validator-node/using-azure.md [hunk 3]
```diff
@@ -150,7 +150,12 @@ This will download all the Terraform dependencies for you, in the `.terraform` f
 
     This will create two YAML files in the `~/$WORKSPACE/$USERNAME` directory: `owner.yaml` and `operator.yaml`. 
 
-12. Download the genesis blob and waypoint for the network you want to connect to. See [Node Files](/nodes/node-files.md) for a full list of files you should download and the download commands. 
+12. Download the following files by following the download commands on the [Node Files](/nodes/node-files.md) page:
+    - `genesis.blob`
+    - `waypoint.txt`
+    - `haproxy.cfg`
```
Comment (perryjrandall) [inline developer-docs-site/docs/nodes/validator-node/operator/running-validator-node/using-azure.md:156]: same comment for terraform

File: developer-docs-site/docs/nodes/validator-node/operator/running-validator-node/using-gcp.md [hunk 4]
```diff
@@ -149,7 +149,12 @@ This will download all the Terraform dependencies for you, in the `.terraform` f
 
     This will create two YAML files in the `~/$WORKSPACE/$USERNAME` directory: `owner.yaml` and `operator.yaml`. 
 
-12. Download the genesis blob and waypoint for the network you want to connect to. See [Node Files](/nodes/node-files.md) for a full list of files you should download and the download commands. 
+12. Download the following files by following the download commands on the [Node Files](/nodes/node-files.md) page:
+    - `genesis.blob`
+    - `waypoint.txt`
+    - `haproxy.cfg`
```
Comment (perryjrandall) [inline developer-docs-site/docs/nodes/validator-node/operator/running-validator-node/using-gcp.md:155]: same comment

File: developer-docs-site/docs/nodes/validator-node/operator/running-validator-node/using-docker.md [hunk 5]
```diff
@@ -27,7 +27,14 @@ Docker method has only been tested on Linux, Windows, and Intel macOS. If you ar
     cd ~/$WORKSPACE
     ```
 
-2. Download the validator.yaml, docker-compose.yaml and blocked.ips configuration files into this directory. See [Node Files](/nodes/node-files.md) for a full list of files you should download and the download commands.  
+2. Download the following files by following the download commands on the [Node Files](/nodes/node-files.md) page:
+    - `validator.yaml`
+    - `docker-compose.yaml`
+    - `docker-compose-src.yaml`
```
Comment (perryjrandall) [inline developer-docs-site/docs/nodes/validator-node/operator/running-validator-node/using-docker.md:33]: we dont need the docker-compose-src here

**Review by perryjrandall — COMMENTED** [review]
soclose

**Review by perryjrandall — APPROVED** [review]
LFG

**Pull request merged**